server/index.js

```diff
@@ -69,7 +76,18 @@ app.get('*', (req, res) => {
     `)
 })
 
-app.listen(process.env.PORT, () => {
-  // eslint-disable-next-line no-console
-  console.log(`Cardano wallet app listening on ${process.env.PORT}!`)
-})
+/*
+* To run server in secure mode, you need to set ADALITE_ENABLE_HTTPS to true and
+* ADALITE_SERVER_URL to 'https://localhost:3000'
+*/
+if (process.env.ADALITE_ENABLE_HTTPS === 'true') {
```

Comment:
why not simply `if (process.env...) app = https.createServer(options, app)` ? This would save us logic duplication.